workflow/scripts/intersection.py

```diff
@@ -157,26 +155,25 @@ def write_empty_files(columns, outputs_path):
     tqdm.pandas()
     try:
         network_edges_path = snakemake.input['network']
-        attrs = snakemake.config["edge_attrs"]
         hazard_dir = snakemake.input['tifs']
         output_path = snakemake.output['geoparquet']
     except NameError:
-        print(sys.argv)
-        (
-            network_edges_path,
-            attrs,
-            hazard_dir,
-            output_path,
-        ) = sys.argv[1:]
-
-    attrs = attrs.split(",")
+        # print(sys.argv)
+        # (
+        #     network_edges_path,
+        #     hazard_dir,
+        #     output_path,
+        # ) = sys.argv[1:]
+        network_edges_path = '../../results/geoparquet/tanzania-mini_filter-highway-core/slice-2.geoparquet'
+        output_path = '../../results/test.geoparquet'
+        hazard_dir = '../../results/input/hazard-aqueduct-river/tanzania-mini'
```

Comment:
Is this a testing/debugging change that has crept in?

Reply:
Good spot!

Reply:
This is now fixed and tests have been updated. The intersection tests takes ~45s to run, nearly half the total test time -- I can't remember if it always did that, but adding extra info to geoparquet input files shouldn't cause that process to take longer than before.